djangocms_versioning/managers.py

```diff
@@ -9,6 +9,4 @@ def get_queryset(self):
         """Limit query to published content
         """
         queryset = super().get_queryset()
-        if not self.versioning_enabled:
-            return queryset
-        return queryset.filter(versions__state=PUBLISHED)
+        return queryset
```

Comment:
It would be great if we could find out the reason for limiting the queryset in this way. From the code that I've read so far I haven't seen a good reason to keep it this way.